README.md

```diff
@@ -0,0 +1,26 @@
+# Field Formatter
+
+[Field Formatter](https://www.drupal.org/project/field_formatter) module adds formatters for entity reference fields to output only a specific field or field value using a specific formatter w/o needing to configure media view mode and configuring it to display properly. This module provides two formatters: *Field formatter with inline settings* and *Field formatter from view display*.
+
+## Installation
+
+1. Download [Field Formatter](https://www.drupal.org/project/field_formatter) from [GitHub](https://github.com/drupal-media/field_formatter).
```

Comment:
We sync the code to drupal.org. People should download from there most of the time. 

We can add DEVELOPING.md, which will explain that we also accept PRs on GitHub. See example: https://github.com/drupal-media/media_entity/blob/8.x-1.x/DEVELOPING.md